### PR TITLE
Add Salut à Toi news

### PR DESCRIPTION
**Pull request by goffi-contrib**

### CODE REVIEW

**Comment by Echolon** [comment]
Thank your @goffi-contrib !

Can you maybe put that into one textblock? Or use bullet points with a general introduction on the topic?

**Comment by goffi-contrib** [comment]
> Thank your @goffi-contrib !
> 
> Can you maybe put that into one textblock? Or use bullet points with a general introduction on the topic?

sure, I'll update it.

**Comment by goffi-contrib** [comment]
@Echolon done + added a screenshot.

File: content/posts/newsletter/2020-08-08.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ You do not need to use a formal tone, be passionate about why this motivates you
 
 Monal is organizing translations for the app on [Weblate](https://hosted.weblate.org/engage/monal/) now. 27 languages are waiting for excellent and native speakers!
 
+There have been UI and file sharing improvements on the "[Salut à Toi](https://salut-a-toi.org/)" project:
+
+- Libervia, the web frontend, has seen a major UI revamp, with a new theme based on Bulma CSS framework. The [W31 progress note](https://www.goffi.org/b/n9nzn3ySF7Lv4pYBpTqYtP/progress-note) also explains how Nunjucks is used in the browser to use the same templates as in the backend (Jinja 2).
+- SàT's included file sharing component now implements [HTTP File Upload](https://xmpp.org/extensions/xep-0363.html), which completes the [Jingle File Transfer](https://xmpp.org/extensions/xep-0234.html) method. Files uploaded to this component from any XMPP client will appear in SàT's file sharing UI, and can be deleted from there (or with [Ad-Hoc Commands](https://xmpp.org/extensions/xep-0050.html)).
```
Comment (pmaziere) [inline content/posts/newsletter/2020-08-08.md:40]: to be consistent across the newsletter, I suggest:
`[HTTP File Upload](https://xmpp.org/extensions/xep-0363.html)` => `[XEP-0363](https://xmpp.org/extensions/xep-0363.html) (HTTP File Upload)`
`[Jingle File Transfer](https://xmpp.org/extensions/xep-0234.html)` => `[XEP-0234](https://xmpp.org/extensions/xep-0234.html) (Jingle File Transfer)`
`[Ad-Hoc Commands](https://xmpp.org/extensions/xep-0050.html)` => `[XEP-0050](https://xmpp.org/extensions/xep-0050.html) (Ad-Hoc Commands)`

File: content/posts/newsletter/2020-08-08.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ You do not need to use a formal tone, be passionate about why this motivates you
...
+- Libervia's photo album has been greatly improved with a mobile and desktop friendly slideshow, easy way to upload files from browser, and integration of videos. Work is currently in progress to make SàT/Libervia a good fit for an XMPP based familial social network, with a release expected around the end of September or beginning of October.
+![screenshot of Libervia (SàT) photo album with new theme](https://upload.goffi.org/upload/7qWd9IgSf7OIYj-a/sat_libervia_new_theme_photos.png)
```
Comment (pmaziere) [inline content/posts/newsletter/2020-08-08.md:42]: TODO: upload image to xsf github relevant location

**Comment by Echolon** [comment]
@goffi-contrib can you work on the requests? We have a review deadline after tomorrow :/

File: content/posts/newsletter/2020-08-08.md (hunk 2 above)
Comment (goffi-contrib) [reply content/posts/newsletter/2020-08-08.md:42]: I guess there is nothing I can do here, can I?

**Comment by goffi-contrib** [comment]
@pmaziere @Echolon modifications done.

File: content/posts/newsletter/2020-08-08.md (hunk 2 above)
Comment (Echolon) [reply content/posts/newsletter/2020-08-08.md:42]: Yep, I'll do it

**Comment by Echolon** [comment]
Ahh, sorry - can you just replace the picture with:

PLACEHOLDER: https://upload.goffi.org/upload/7qWd9IgSf7OIYj-a/sat_libervia_new_theme_photos.png

**Comment by Echolon** [comment]
@goffi-contrib all good, I just added it myself! Thanks for your contribution!

3664125

**Comment by goffi-contrib** [comment]
@Echolon thanks!

**Pull request closed without merging**